Sources/FBLPromises/include/FBLPromise.h

```diff
@@ -39,14 +39,14 @@ NS_ASSUME_NONNULL_BEGIN
  @param resolution An object to resolve the promise with: either a value or an error.
  @return A new resolved promise.
  */
-+ (instancetype)resolvedWith:(nullable id)resolution NS_REFINED_FOR_SWIFT;
++ (instancetype)resolvedWith:(nullable Value)resolution NS_REFINED_FOR_SWIFT;
```

Comment:
This type indicates only a value can be passed, but we document that a value or an error is allowed.